pkg/pot/lex.go

```diff
@@ -197,9 +82,9 @@ func Lex(r io.Reader) ([]Token, error) {
 			continue
 		}
 
-		token, err := parseLine(line, &tokens)
+		token, err := parseLine(line, tokens)
 		if err != nil {
-			return nil, fmt.Errorf("parse line %d: %w", lineNumber, err)
+			return nil, fmt.Errorf("parse line \"%s\": %w", line, err)
```

Comment:
alternatively, easier to read with backquotes

```suggestion
			return nil, fmt.Errorf(`parse line "%s": %w`, line, err)
```